src/Common/src/System/Security/Cryptography/CertificateValidation.Unix.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+using System.Globalization;
+using System.Net;
+using System.Net.Security;
+
+namespace System.Security.Cryptography.X509Certificates
```

Comment:
This doesn't seems like it really belongs in this namespace (and include path), due to layering problems.  `SslPolicyErrors` is a System.Net.Security enum, so it's really a System.Net.Security-ism more than an X509-ism.

So none of the cryptography libraries could include this thing from the crypto namespace.